lib/_debugger.js

```diff
@@ -254,9 +254,18 @@ Client.prototype._onResponse = function(res) {
 
 
 Client.prototype.req = function(req, cb) {
-  this.write(this.protocol.serialize(req));
-  cb.request_seq = req.seq;
-  this._reqCallbacks.push(cb);
+  var self = this;
+  self.write(this.protocol.serialize(req));
+  var timer = setTimeout(function() {
+    console.log('request timeout');
+    self.emit('close');
+  }, 3000);
+  var _cb = function(err, res) {
```

Comment:
What's difference between them? Actually I just wanna keep consistent with the line 259 which is also an assignment.